cmd/prom-aggregation-gateway/main.go

```diff
@@ -259,12 +301,15 @@ func (a *aggate) handler(w http.ResponseWriter, r *http.Request) {
 }
 
 func main() {
-	listen := flag.String("listen", ":80", "Address and port to listen on.")
+	listen := flag.String("listen", ":8080", "Address and port to listen on.")
 	cors := flag.String("cors", "*", "The 'Access-Control-Allow-Origin' value to be returned.")
 	pushPath := flag.String("push-path", "/metrics/", "HTTP path to accept pushed metrics.")
+	timeToLiveMs := flag.Int64("time-to-live-ms", 3600000, "How long unmerged metrics will live, in milliseconds (default 1h)")
```

Comment:
This is a little user-hostile; `flag.Duration` would allow specifying as `1h`, `10m`, etc.